backend/routes/employeeRouters.ts

```diff
@@ -1,39 +1,20 @@
 import express, { NextFunction, Request, response, Response } from "express";
 const employeeRouter = express.Router();
 
-import {
-  EmployeeDataClause,
-  EmployeeWhereClause,
-} from "../interfaces/employeeInterface";
+import { EmployeeInsert } from "../interfaces/employeeInterface";
 import * as employeeController from "../controllers/employeeControllers";
 
-type WhereClauseBuilderFunc = (query: any) => EmployeeWhereClause;
-const _whereClauseBuilder: WhereClauseBuilderFunc = (query: any) => {
-  const whereClause: EmployeeWhereClause = {
-    id: query.id,
-    email: query.email,
-    password: query.password,
-    firstName: query.firstName,
-    lastName: query.lastName,
-    pronoun: query.pronoun,
-    position: query.position,
-    isManager: Boolean(query.isManager),
-  };
-
-  return whereClause;
-};
-
-type DataClauseBuilderFunc = (body: any) => EmployeeDataClause;
-const _dataClauseBuilder: DataClauseBuilderFunc = (body: any) => {
-  const whereClause: EmployeeDataClause = {
-    id: body.id,
-    email: body.email,
-    password: body.password,
-    firstName: body.firstName,
-    lastName: body.lastName,
-    pronoun: body.pronoun,
-    position: body.position,
-    isManager: Boolean(body.isManager),
+type EmployeeInsertQueryBuilder = (body: any) => EmployeeInsert;
```

Comment:
EmployeeQueryBuilder, has nothing to do with EmployeeInsert

Reply:
Probably doesn't need a type name anyway. Just declare the functions with type annotations

Reply:
```typescript
// Idiomatic
function dataClauseBuilder(body: any): EmployeeInsert {
  return {
    //
  };
}
// If you need an arrow function (this, scope, etc)
const dataClauseBuilderArrow = (body: any): EmployeeInsert => {
  return {
    //
  };
}
```